src/Providers.tsx

```diff
@@ -1,24 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
We can clean the React import as it is not useful anymore in Next (following the new React auto import made possible by bundlers).

```suggestion
import { useEffect, useState } from 'react';
```

Reply:
I will not do it in this PR for 2 reasons:
1. If we choose to remove the "React" import, we should do it in a separate PR that will remove all "React" import in all the code base for consistency
2. For now, I'm not ok to remove the "React" import because not all bundlers are ready for this. And if we need to change the bundler away from Next or if someone need to copy/paste some code, it might not work because of this.